magpie-check-inputs

```diff
@@ -1332,4 +1335,65 @@ then
     fi
 fi
 
+#
+# Check Alluxio Inputs
+#
+
+if [ "${ALLUXIO_SETUP}" == "yes" ]
+then
+
+    __Magpie_check_must_be_set "JAVA_HOME" "for Alluxio"
+
+    __Magpie_check_must_be_set "ALLUXIO_VERSION" "to run Alluxio"
+    __Magpie_check_if_version_format_correct "ALLUXIO_VERSION" "to run Alluxio"
+
+    __Magpie_check_must_be_set_and_is_directory "ALLUXIO_HOME" "to run Alluxio"
+
+    __Magpie_check_must_be_set "ALLUXIO_WORKER_ON_MASTER" "for Alluxio"
+    __Magpie_check_if_set_is_yes_or_no "ALLUXIO_WORKER_ON_MASTER"
+
+    __Magpie_check_must_be_set "ALLUXIO_LOCAL_DIR" "to run Alluxio"
+
+    __Magpie_check_must_be_set "ALLUXIO_UNDER_FS_DIR" "to run Alluxio"
+
+    __Magpie_check_must_be_set "ALLUXIO_DATA_CLEAR" "for Alluxio"
+    __Magpie_check_if_set_is_yes_or_no "ALLUXIO_DATA_CLEAR"
+
+    __Magpie_check_must_be_set "ALLUXIO_WORKER_MEMORY_SIZE" "to run Alluxio"
```

Comment:
in your documentation in README it says ALLUXIO_WORKER_MEMORY_SIZE defaults to 2/3 of memory?  Is that wrong?  Seems to not match the fact that ALLUXIO_WORKER_MEMORY_SIZE must be set.

Reply:
> By default, Alluxio allocates 2/3 of total system memory, which will not be a valid setting for Magpie run when other components are running on the same hosts as Alluxio, adjust appropriately given memory requirements of other services.

In this quote, I meant the default Alluxio that works standalone, not in the Magpie stack (abstract). 2/3 of memory won't be a reasonable setting if we have spark working on the same hosts as Alluxio (but maybe would be reasonable if Alluxio would be the only software in Magpie stack). If it is not clear, for sure I have to rephrase that. Maybe we should remove this 2/3 number to avoid misunderstanding. Or implement to support this 2/3 default value. But not sure if this should be used there. I would rather like to manually and intentionally set it here. Not sure.

Let me know what do you think @chu11.

Reply:
I would say lets just remove the 2/3 comment.  I would just put something like "Set the amount of memory each alluxio worker will use" or whatever is appropriate.  Thanks!

Reply:
I changed that to:

`Recommended memory size is 2/3 of total system memory, but it will not be a valid setting for Magpie run when other components are running on the same hosts as Alluxio, adjust appropriately given memory requirements of other services.`

Reply:
@chu11 changed that to `Set the amount of memory each Alluxio worker will use. Memory would be used as the first tier of storage. The more memory would Alluxio have, the bigger first tier storage would be.`

Reply:
How about we just add a small "Be mindful of the memory requirements of other services (e.g. Spark)."